poc-step-function/step-functions/standard-etl/standard-etl.json

```diff
@@ -0,0 +1,133 @@
+      "Default": "Process run groups in serial"
+    },
+    "Process run groups in serial": {
+      "Type": "Map",
+      "MaxConcurrency": 1,
+      "InputPath": "$",
+      "Next": "Finish",
+      "ItemsPath": "$.orderedRunGroups",
+      "Parameters": {
+        "RunGroupIndex.$": "$$.Map.Item.Index",
+        "RunGroupArray.$": "$$.Map.Item.Value"
+      },
+      "Iterator": {
+        "StartAt": "Run jobs in run group in parallel",
+        "States": {
+          "Run jobs in run group in parallel": {
+            "Type": "Map",
+            "MaxConcurrency": 10,
+            "InputPath": "$",
+            "ItemsPath": "$.RunGroupArray",
+            "Parameters": {
+              "ETLJobIndex.$": "$$.Map.Item.Index",
+              "ETLJob.$": "$$.Map.Item.Value"
+            },
+            "Iterator": {
+              "StartAt": "Run ETL job tasks in serial",
+              "States": {
+                "Run ETL job tasks in serial": {
+                  "Type": "Map",
+                  "MaxConcurrency": 1,
+                  "InputPath": "$",
+                  "ItemsPath": "$.ETLJob.tasks",
+                  "Parameters": {
+                    "TaskIndex.$": "$$.Map.Item.Index",
+                    "Task.$": "$$.Map.Item.Value"
+                  },
+                  "Iterator": {
+                    "StartAt": "Choose ETL Task Type",
+                    "States": {
+                      "Choose ETL Task Type": {
+                        "Type": "Choice",
+                        "InputPath": "$",
+                        "Choices": [
+                          {
+                            "Variable": "$.Task.type",
+                            "StringEquals": "noop",
+                            "Next": "Run Noop"
+                          },
+                          {
+                            "Variable": "$.Task.type",
+                            "StringEquals": "sql",
+                            "Next": "Run Sql"
+                          }
+                        ],
+                        "Default": "Invalid Job Type"
+                      },
+                      "Run Noop": {
+                        "Type": "Task",
+                        "Resource": "arn:aws:states:::lambda:invoke",
+                        "InputPath": "$",
+                        "ResultPath": null,
+                        "Parameters": {
+                          "FunctionName": "CoaNoop",
+                          "Payload": {
+                            "Task.$": "$.Task"
+                          }
+                        },
+                        "End": true
+                      },
+                      "Run Sql": {
+                        "Type": "Task",
+                        "Resource": "arn:aws:states:::lambda:invoke",
+                        "InputPath": "$",
+                        "ResultPath": null,
+                        "Parameters": {
+                          "FunctionName": "CoaNoop",
```

Comment:
Instead of creating a separate function for SQL right now, this example just runs the `CoaNoop` job as a placeholder.